app/admin/dashboard.rb

```diff
@@ -36,7 +36,7 @@
       column do
         panel "ActiveAdmin Demo" do
           div do
-            render('/admin/sidebar_links', :model => 'dashboards')
+            render('/admin/sidebar_links', :model => 'dashboard')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.